packages/bson-transpilers/symbols/javascript/templates.yaml

```diff
@@ -220,24 +220,12 @@ Templates:
     CodeCodeArgsTemplate: &CodeCodeArgsTemplate null
     CodeScopeTemplate: &CodeScopeTemplate null
     CodeScopeArgsTemplate: &CodeScopeArgsTemplate null
-    ObjectIdToStringTemplate: &ObjectIdToStringTemplate !!js/function >
```

Comment:
this was not necessary, only breaks some of the transformations that are correctly handled by default